packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -100,7 +100,13 @@
           </oc-button>
         </div>
         <div>
-          <oc-button v-if="saving" variation="primary" appearance="filled" disabled>
+          <oc-button
+            v-if="saving"
+            id="oc-saving-public-link-button"
```

Comment:
Please check the `id` for the button below and use something that is similiar since they have a very similar use case